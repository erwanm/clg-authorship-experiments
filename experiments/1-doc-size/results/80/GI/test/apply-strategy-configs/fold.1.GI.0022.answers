1903-us-tjoas 1854-hbs-smofl	0.529004
1895-wta-alae 1906-mt-cfma	0.556361
1891-fmc-ttf 1881-haj-fcbtp	0.644545
1872-haj-ptf 1906-rwc-ttolp	0.591429
1898-hj-ttm 1879-hj-h	0.579397
1874-wta-tcw 1900-haj-adoh	0.531757
1878-hj-te 1882-lma-ajsb-v6	0.488521
1883-hj-rh 1898-hj-itc	0.701873
1885-hj-gr 1910-us-sts	0.442566
1914-us-sm 1897-ewaoc-tdof	0.12624
1896-espw-cfal 1887-espw-jtf	0.473268
1887-espw-tgb 1900-espw-wtg	0.477585
1894-cdw-tgh 1907-fmc-tdr	0.381518
1900-fmc-itpotk 1905-fmc-fm	0.500734
1889-fmc-g 1898-haj-tyb	0.089194
1890-hj-ttm 1904-hj-tgb	0.611071
1878-hbs-pp 1870-hbs-tbc	0.498066
1901-us-km 1921-us-tbol	0.444773
1883-hj-rh-v2 1872-wta-lb	0.089194
1866-haj-ccc-ia 1871-hbs-mwai	0.526649
1880-hj-c 1885-hj-altof	0.239715
1898-fmc-ari-v2 1893-fmc-kl-v2	0.575072
1918-rwc-tlg 1863-wta-isao	0.395347
1884-ab-sbts 1898-us-apom	0.422646
1889-cfw-jl 1915-rwc-p	0.532551
1872-mt-ri 1895-haj-tdw	0.089194
1902-fmc-c 1883-fmc-tl	0.634775
1859-hbs-tmw 1887-fmc-pp	0.583022
1884-wta-Sac 1882-wta-aa	0.677767
1871-haj-tt 1876-wta-vao	0.549421
1913-es-tm 1911-rwc-tgr	0.200219
1905-es-tpoe 1907-es-tlotg	0.556632
1899-haj-mmv 1866-haj-hf	0.089194
1891-fmc-do 1876-cdw-itl	0.23108
1892-fmc-pg 1883-fmc-dc	0.478175
1917-ab-bn 1922-us-tcmc	0.47928
1889-cdw-sths 1869-espw-mwag	0.530989
1902-hj-twotd 1896-hj-tsop	0.688789
1888-haj-bb 1899-haj-ra	0.597357
1913-ew-tcotc 1868-haj-rd	0.630413
1868-espw-tga 1919-rwc-tsos	0.57865
1900-haj-ainy 1919-ew-m	0.369361
1881-mt-tpatp 1899-haj-aoatb	0.089194
1891-fmc-k 1893-fmc-tr	0.543802
1881-hj-tpoal 1890-fmc-acmr	0.531266
1869-haj-mtmb 1903-haj-bba-ph	0.685532
1885-fmc-atoalp 1905-fmc-wso	0.089194
1868-wta-mob 1870-wta-faf	0.710347
1871-hbs-pawt 1904-rwc-isotu	0.315109
1875-cfw-cn 1864-lma-m	0.589022
1914-mh+-mhce-nf 1908-fmc-tp	0.540746
1893-fmc-md 1895-fmc-ajs	0.536988
1864-wta-tsb 1883-wta-sh	0.594356
1900-rwc-taoamm 1910-rwc-tgm	0.663147
1903-fmc-thor 1887-fmc-s	0.377747
1884-mt-taohf 1874-mt-lotm	0.308927
1907-espw-tldup 1871-espw-tsp	0.591083
1866-espw-gcj 1882-haj-bn	0.500037
1883-haj-tbb 1888-es-ttatv	0.555416
1908-fmc-s 1908-fmc-tws	0.617577
1889-wta-wtel 1911-rwc-tcl	0.465387
1904-rwc-sm 1886-hj-pc	0.252381
1918-us-jh 1897-ew-tdof	0.529834
1872-lma-ss 1867-lma-tagomtt	0.61599
1910-ew-tomag 1889-es-atg	0.47794
1889-espw-tsfi 1895-espw-tsojc	0.632248
1905-haj-fftf 1870-haj-rar	0.686705
1882-espw-dz 1909-mt-isd	0.480908
1907-ew-tfott 1896-hj-toh	0.384719
1868-wta-dad 1885-wta-sts	0.702245
1909-rwc-tdm 1897-ab-tdohy	0.513455
1891-haj-tetb 1868-haj-faf	0.631843
1893-haj-dtn 1866-haj-tcw	0.566177
1869-wta-dtr 1865-wta-waw	0.535443
1891-cdw-oi 1873-hbs-wish	0.579742
1879-hj-dm 1903-hj-ta	0.519425
1874-cfw-tac 1882-fmc-mi	0.673157
1893-espw-dm 1896-ab-boat	0.196917
1896-espw-tsasa 1894-espw-asl	0.600218
1879-haj-tym 1869-haj-pjb	0.362469
1920-us-1tsoap 1913-us-dg	0.480174
1917-es-ow-sh 1873-hbs-pl	0.588624
1855-wta-aa 1873-wta-tyc	0.642423
1879-haj-ttb 1894-fmc-wg	0.407726
1898-fmc-vc 1895-fmc-t	0.471702
1867-espw-gyatgc 1901-espw-a	0.495705
1896-fmc-c 1885-fmc-zamc	0.549673
1855-hbs-tmf 1866-espw-gb	0.408275
1897-fmc-aroy 1884-fmc-aap	0.468936
1883-espw-btg 1911-hj-to	0.353038
1862-hbs-tpooi 1875-hbs-waon	0.705436
1883-fmc-ars 1885-fmc-tcotk	0.6781
1886-hbs-tsw 1854-hbs-smofl-v2	0.585099
1903-ew-ivatg 1878-cdw-hikab	0.625157
1858-wta-pap 1897-haj-fof	0.483148
1902-hj-twotd-v2 1888-hj-tr	0.436716
1871-hbs-ofs 1888-fmc-si	0.535319
1907-ab-rm 1901-hj-tsf	0.623706
1882-haj-ffbts 1893-haj-ianwatgfoa	0.435005
1908-rwc-tfl 1910-rwc-ap	0.089194
