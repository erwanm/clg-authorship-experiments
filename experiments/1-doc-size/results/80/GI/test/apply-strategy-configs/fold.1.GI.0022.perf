0.396000	0.660000	0.600000
