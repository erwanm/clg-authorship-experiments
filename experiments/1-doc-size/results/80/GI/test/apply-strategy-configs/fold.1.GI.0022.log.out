apply-config.sh: applying config '/tmp/expe1/80/process/selected-meta-configs/fold.1.strategy-as-meta.GI.0022.model/strategy-configs/fold.1.GI.0022' to cases '/home/moreaue/dump/expe1//results/80/GI/test/apply-strategy-configs/only-cases.tmp.1JoA42Xv'; results written to '/home/moreaue/dump/expe1//results/80/GI/test/apply-strategy-configs/fold.1.GI.0022'
apply-config.sh: done.
